tools/libdeflate/Makefile

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (C) 2022 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libdeflate
+PKG_VERSION:=1.15
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE_URL:=https://github.com/ebiggers/libdeflate.git
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_VERSION:=020133854ff73b8506fe59f92a9b5b622d360716
```

Comment:
You can replace this with v1.15

Reply:
Can I replace this also with `$(PKG_VERSION)` not duplicate that bit twice?

Reply:
Yes